app/build.gradle

```diff
@@ -53,5 +53,5 @@ dependencies {
     implementation libs.lifecycle
```

Comment:
Please add the pr description

Reply:
will do yesterday, I missed that part sorry :(